storage/innobase/btr/btr0btr.cc

```diff
@@ -844,14 +844,20 @@ static rec_offs *btr_page_get_parent(rec_offs *offsets, mem_heap_t *heap,
         if (page_cur_search_with_match(tuple, PAGE_CUR_LE, &up_match,
                                        &low_match, &cursor->page_cur,
                                        nullptr))
+        {
+          sql_print_information("InnoDB: btr_page_get_parent 1");
           return nullptr;
+        }
         offsets= rec_get_offsets(cursor->page_cur.rec, index, offsets, 0,
                                  ULINT_UNDEFINED, &heap);
         p= btr_node_ptr_get_child_page_no(cursor->page_cur.rec, offsets);
         if (p != page_no)
         {
           if (btr_page_get_level(block->page.frame) == level)
+          {
+            sql_print_information("InnoDB: btr_page_get_parent 2");
```

Comment:
I think it shouldn't be part of the final patch.

Reply:
You are right, the entire commit 3dc8ab782d7489a2e5a9f6bdc3295a3a835ed1e6 was there for testing purposes only (and successful at that; see [MDEV-32174](https://jira.mariadb.org/browse/MDEV-32174). It will be dropped in the final version.